src/main.go

```diff
@@ -0,0 +1,241 @@
+}
+
+func (c *WorkflowNotificationCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options]
+
+  The chat command sends messages to Chat spaces.
+`
+}
+
+func (c *WorkflowNotificationCommand) Flags() *cli.FlagSet {
+	set := cli.NewFlagSet()
+
+	f := set.NewSection("Chat space options")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "webhook_url",
+		Aliases: []string{"w"},
+		Example: "https://chat.goog...",
+		Default: "",
+		Target:  &c.flagWebhookUrl,
+		Usage:   `Webhook URL from google chat`,
+	})
+
+	return set
+}
+
+func (c *WorkflowNotificationCommand) Run(ctx context.Context, args []string) error {
+	f := c.Flags()
+	if err := f.Parse(args); err != nil {
+		return fmt.Errorf("failed to parse flags: %w", err)
+	}
+
+	args = f.Args()
+	if len(args) != 0 {
+		return fmt.Errorf("expected 0 arguments, got %q", args)
+	}
+
+	ghJsonStr := os.Getenv(githubContextEnv)
+	if ghJsonStr == "" {
+		return fmt.Errorf("environment var %s not set", githubContextEnv)
+	}
+	jobJsonStr := os.Getenv(jobContextEnv)
+	if jobJsonStr == "" {
+		return fmt.Errorf("environment var %s not set", jobContextEnv)
+	}
+
+	ghJson := map[string]any{}
+	jobJson := map[string]any{}
+	if err := json.Unmarshal([]byte(ghJsonStr), &ghJson); err != nil {
+		return fmt.Errorf("failed unmarshaling %s: %w", githubContextEnv, err)
+	}
+	if err := json.Unmarshal([]byte(jobJsonStr), &jobJson); err != nil {
+		return fmt.Errorf("failed unmarshaling %s: %w", jobContextEnv, err)
+	}
+
+	b, err := messageBody(ghJson, jobJson)
+	if err != nil {
+		return fmt.Errorf("failed to generate message body: %w", err)
+	}
+
+	url := c.flagWebhookUrl
+	fmt.Println("url: ", url)
+
+	request, err := http.NewRequest("POST", url, bytes.NewBuffer(b))
+	if err != nil {
+		return fmt.Errorf("creating http request failed: %w", err)
+	}
+	resp, err := http.DefaultClient.Do(request)
+	if err != nil {
+		return fmt.Errorf("sending http request failed: %w", err)
+	}
+	fmt.Println("resp: ", resp)
+	defer resp.Body.Close()
+
+	return nil
+}
+
+func main() {
+	if err := realMain(); err != nil {
+		fmt.Fprintln(os.Stderr, err.Error())
+		os.Exit(1)
+	}
+}
+
+func realMain() error {
+	rootCmd := func() cli.Command {
+		return &cli.RootCommand{
+			Name: "send-google-chat-webhook",
+			Commands: map[string]cli.CommandFactory{
+				"chat": func() cli.Command {
+					return &cli.RootCommand{
+						Name:        "workflownotification",
+						Description: "notification for workflow",
+						Commands: map[string]cli.CommandFactory{
+							"workflownotification": func() cli.Command {
+								return &WorkflowNotificationCommand{}
+							},
+						},
+					}
+				},
+			},
+		}
+	}
+
+	cmd := rootCmd()
+	// Help output is written to stderr by default. Redirect to stdout so the
+	// "Output" assertion works.
+	cmd.SetStderr(os.Stdout)
+
+	ctx := context.Background()
+	err := cmd.Run(ctx, os.Args[1:])
+	if err != nil {
+		return fmt.Errorf("failed to run command: %w", err)
+	}
+
+	return nil
```

Comment:
I think you just want:

```suggestion
	return rootCmd().Run(ctx, os.Args[1:]) //nolint:wrapcheck // Want passthrough
```